700-quarkus-infinispan-grpc-kafka/src/main/java/io/quarkus/qe/books/Book.java

```diff
@@ -0,0 +1,71 @@
+package io.quarkus.qe.books;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotBlank;
+
+import org.infinispan.protostream.annotations.ProtoFactory;
+import org.infinispan.protostream.annotations.ProtoField;
+
+public class Book {
+
+    private String title;
+    private String description;
+    private int publicationYear;
+
+    public Book() {
+
+    }
```

Comment:
This empty constructor is necessary to use this class in the rest endpoint (not a good practice though).